Web3/Classes/Core/Transaction/EthereumTransaction.swift

```diff
@@ -17,7 +17,7 @@ public struct EthereumTransaction: Codable {
     public var gasPrice: EthereumQuantity?
     
     /// Gas limit provided
-    public var gasLimit: EthereumQuantity?
+    public var gas: EthereumQuantity?
```

Comment:
I changed this because the json object generated from the Codable implementation here was using `gasLimit` as the key, and the JSON-RPC spec requires it to be `gas`. That comes into play when doing `sendTransaction`, which was added to the Eth module in this PR. I thought this would be a simpler fix than implementing custom Codable api. Would you prefer to keep it as gasLimit?

Reply:
No it's fine. I just wanted to know the reason.